splash/tests/mockserver.py

```diff
@@ -15,7 +17,8 @@
 
 _REQUIRED = object()
 
-def getarg(request, name, default=_REQUIRED, type=str):
+
+def getarg(request, name, default=_REQUIRED, type=bytes):
```

Comment:
Can code be simplified if we keep using `str` here instead of `bytes`? I see results of `getarg` are often decoded.